data/human/culture conversations.txt

```diff
@@ -1101,3 +1101,48 @@ mission "Little Vienna in New Austria"
 			label backtoship
 			`	With the atmosphere steeped in nostalgia for a distant city on a distant world long past, you return to your ship.`
 
+mission "Ramadan on Farpoint"
+	minor
+	source "Farpoint"
+	to offer
+		random < 70
+		month == 3
+	on offer
+		conversation
+			`The red desert of <origin> stretches beyond and beneath the spaceport, creating the illusion that the city is perched on a sea of cooling magma.`
+			`	Carved into one of the cliff faces is a small enclosure with a glass wall facing the main spaceport. Inside, a group of men wearing white robes goes through a methodical series of motions, falling to the ground, kneeling, and standing, while chanting in mellifluous Arabic.`
+			`	A few minutes after you begin watching, the sun goes down. Almost immediately, the group scatters, going to little recesses within the walls. Each contains a date and a small cup of water.`
+			`	Another group of adults, wearing black robes, comes out of an interior room. They too go to alcoves in the walls, consuming the dates. One of them notices you watching, and comes toward you with a piece of fruit and a small cup of water.`
+			choice
+				`	(Accept the food.)`
+				`	(Shake your head and leave.)`
+					decline
+			`	You are drawn into the crowd. Soon afterwards, they move to another room, which is filled with a huge table bearing a literal feast. The group sits down.`
+			`	Some while later, you are led back to your ship, after being wished a Happy Ramadan.`
+				decline
+
+mission "Easter Celebrations"
+	minor
+	source
+		attributes "religious"
+		government "Deep" "Free Worlds" "Independent" "Neutral" "Pirate" "Republic" "Syndicate"
+	to offer
+		day == 20
+		month == 4
+	on offer
+		conversation
+			`During a walk through the spaceport, you observe a small white church. Written on the door in large letters are the words "Stop by for a celebration of the Risen God!"`
+			choice
+				`	(Step in.)`
+				`	(Move on.)`
+					decline
+			`	Inside the building, a group is singing with deafening loudness. On the stage at the front of the room, a man strums a guitar; behind him a lady softly plays the piano. A large screen displays lyrics to the current song, which you note is "Christ Our Lord has Risen Again."`
+			choice
+				`	(Head out.)`
+					goto headout
+				`	(Stay a little bit longer.)`
+			`	After a while longer, you exit the room. As you do so, someone hands you a ticket to a concert next week on the other side of the planet. It's too bad you won't be able to attend.`
```

Comment:
I feel like you should be able to sit down and enjoy the music. ~~Hymns can be annoyingly catchy.~~